test/langtools/tools/javac/util/StringUtilsTest.java

```diff
@@ -53,6 +55,57 @@ void run() throws Exception {
         assertEquals(2, StringUtils.indexOfIgnoreCase("  lookFor", "lookfor"));
         assertEquals(11, StringUtils.indexOfIgnoreCase("  lookFor  LOOKfor", "lookfor", 11));
         assertEquals(2, StringUtils.indexOfIgnoreCase("\u0130\u0130lookFor", "lookfor"));
+
+        //verify Damerau-Levenshtein
+
+        assertEquals(3, DamerauLevenshteinDistance.of("kitten", "sitting"));
+        // note that the unrestricted Damerau-Levenshtein distance would be 3, not 2:
```

Comment:
I think the "unrestricted" in this comment should instead be "restricted".

Reply:
Fixed; thanks!